osu.Game.Rulesets.Osu/Mods/OsuModTransform.cs

```diff
@@ -30,38 +35,92 @@ internal class OsuModTransform : ModWithVisibilityAdjustment
 
         protected override void ApplyNormalVisibilityState(DrawableHitObject hitObject, ArmedState state) => applyTransform(hitObject, state);
 
+        [SettingSource("Movement type", "Alter how circles move into place", 1)]
```

Comment:
What's the `1` at the end there for? This is the only setting for this mod, is it not? Why would `orderPosition` matter?

Reply:
yes its the only setting, my bad, good catch. probably slipped in whilst i was testing something